packages/terra-clinical-result-view/src/ResultView.jsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import IconDocuments from 'terra-icon/lib/icon/IconDocuments';
+import IconComment from 'terra-icon/lib/icon/IconComment';
+import IconModified from 'terra-icon/lib/icon/IconModified';
+import { elementType } from 'react-prop-types';
+import styles from './ResultView.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   *  An array of one or more results.
+   */
+  results: PropTypes.arrayOf(PropTypes.shape({
+    value: PropTypes.number,
+    unit: PropTypes.string,
+    icon: elementType('svg'),
+    color: PropTypes.string,
+  })),
+  /**
+   *  The age display of the result. Does not handle formatting.
+   */
+  timeDisplay: PropTypes.string,
+  /**
+   *  Does the result have related documents. Adds an indicator.
+   */
+  hasDocument: PropTypes.bool,
+  /**
+   *  Has the result been modified. Adds an indicator.
+   */
+  isModified: PropTypes.bool,
+  /**
+   *  Are there comments related to the result. Adds an indicator.
+   */
+  hasComment: PropTypes.bool,
+  /**
+   *  The alignment of the component. Defaults to left.
+   */
+  alignment: PropTypes.oneOf(['left', 'right', 'center']),
+  /**
+   *  Can the result text be truncated. Defaults to true.
+   */
+  isTruncated: PropTypes.bool,
+  /**
+   *  Should the view have padding. Defaults to false.
+   */
+  isPadded: PropTypes.bool,
+};
+
+const defaultProps = {
+  results: [],
+  timeDisplay: null,
+  hasDocument: false,
+  hasComment: false,
+  isModified: false,
+  alignment: 'left',
+  isTruncated: true,
+  isPadded: false,
+};
+
+const resultIndicator = (result) => {
+  let icon = null;
+
+  if (!result.icon) {
+    return null;
+  }
+  icon = React.cloneElement(
+    result.icon,
+    { color: result.color },
+  );
+
+  return <span className={cx('result-view-indicator')} >{icon}</span>;
+};
+
+const resultsDiv = (results, isTruncated) => {
+  if (!results || !results.length) {
+    const noValueDisplay = '--';
+    return <div className={cx('result-view-result-value-text')} >{noValueDisplay}</div>;
+  }
```

Comment:
Creating the variable makes this longer, what are your thoughts on changing this to:

```
if (!results || !results.length) {
  return <div className={cx('result-view-result-value-text')}>{'--'}</div>;
}
```